IsraelHiking.Web/src/scss/common.scss

```diff
@@ -98,19 +98,32 @@ body {
         padding: 0 !important;
         display: block;
         width: v.$app-button-size;
-        height: v.$app-button-size;
+        height: 44px;
 
         & i {
             vertical-align: middle !important;
         }
     }
+    button.active:not(button.active:first-child , .search button.active ){
+        margin-top:7px;
+    }
 }
-
+.side-control.search
+ {
+    button{
+        margin-top:2px;
+    }
+    button.active {
+        margin-top: 2px;
```

Comment:
Is this needed given that every button should have a margin top of 2px?

Reply:
This only adds a margin to the buttons inside .search (the search section in the center with the plus icon) to prevent overlapping of buttons beneath the plus icon after increasing the height of both the top and bottom search bars.

Reply:
I meant button and button.active have the same properties of margin...

Reply:
I have to Improve this selector and removem`button.active` margin.